drivers/multisensor/driver.js

```diff
@@ -59,6 +59,16 @@ class MultisensorDriver extends Homey.Driver {
         .catch(this.error);
     }
 
+    // Set humidity
+    if (signal.data.humidity != undefined) {
+      if (! (homeyDevice.hasCapability("measure_humidity"))) {
+        homeyDevice.addCapability("measure_humidity").catch(this.error);
+      }
+      homeyDevice
+        .setCapabilityValue("measure_humidity", signal.data.humidity)
+        .catch(this.error);
+    }
+
```

Comment:
@ojnordah This adds the capability to the device when a signal that contains humidity is received